portal-web/test/functional/com/liferay/portalweb/macros/headless/blogs/HeadlessBlog.macro

```diff
@@ -30,4 +31,64 @@ definition {
 		return "${entryId}";
 	}
 
+	macro assertCommentERCWithCorrectValue {
+		var actualCommentERCValue = JSONUtil.getWithJSONPath("${response}", "$..['items'][?(@.creator)].externalReferenceCode");
+
+		TestUtils.assertEquals(
+			actual = "${actualCommentERCValue}",
+			expected = "${expectedCommentERCValue}");
+	}
+
+	macro assertCommentTextWithCorrectValue {
+		var actualTextComments = JSONUtil.getWithJSONPath("${blogsPostingId}", "$..['items'][?(@.creator)].text");
+
+		TestUtils.assertEquals(
+			actual = "${actualTextComments}",
+			expected = "${actualTextComments}");
+	}
+
+	macro assertProperNumberOfItems {
```

Comment:
we already have the macro BlogPostingAPI.getNumberOfBlogPostingsCreated(); which returns a number of blogs created. You can then use it directly in the test step, example:

```
task ("And Then the 2 blog postings are visible as created") {
			var actualNumberOfBlogPostings = BlogPostingAPI.getNumberOfBlogPostingsCreated();

			TestUtils.assertEquals(
				actual = "${actualNumberOfBlogPostings}",
				expected = "2");
		}


```